ScheduleLesson/Controllers/ScheduleController.cs

```diff
@@ -17,14 +20,16 @@ public ScheduleController(IScheduleService scheduleService)
         [HttpGet]
         public async Task<ActionResult<Schedule>> GetAllSchedule()
         {
-            List<Schedule> result = await _scheduleService.GetAllSchedule();
+            string? userId = User.FindFirstValue(ClaimTypes.NameIdentifier); // Отримуємо ідентифікатор користувача
```

Comment:
it is better to convert string in the appropriate Id type(int or Guid) and pass it to your service methods